ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ACSReader.scala

```diff
@@ -38,17 +49,43 @@ class ACSReader(
     dispatcher: DbDispatcher,
     queryNonPruned: QueryNonPruned,
     eventStorageBackend: EventStorageBackend,
+    lfValueTranslation: LfValueTranslation,
     metrics: Metrics,
-    executionContext: ExecutionContext,
-) {
+)(implicit executionContext: ExecutionContext) {
```

Comment:
this refactoring could have stayed in a different PR tbh since it's much bigger than the actual core change that's described in this PR

Reply:
A bunch of methods that I moved here from TransactionReader expects an implicit EC. That seemed to be the most convenient way to provide it
(Tried dropping the `implict` part, but then it became a bit messy to pass `executionContext` manually)